### DISCUSSION TITLE
How can I manipulate defaults?

### DISCUSSION THREAD

**Discussion by bozdevs** [swup/swup#902] [General]
I have Swup&Astro integration -> https://github.com/swup/astro#configuration. But I don't change some defaults, https://swup.js.org/options/#defaults

[#configuration](https://github.com/swup/astro#configuration) and [#defaults](https://swup.js.org/options/#defaults) contains different and missing properties.

For example: 
```js
requestHeaders: {
    'X-Requested-With': 'swup',
    'Accept': 'text/html, application/xhtml+xml'
},
```

**Comment by daun** [comment]
The Astro integration is meant as a convenient and simple way of enabling swup in Astro projects. Hence it doesn't cover all of swup's options and isn't meant to. In the readme, you'll find notes and working examples for [advanced usage and customization](https://github.com/swup/astro?tab=readme-ov-file#advanced-usage). Both suggested options should work in your use case — working with a global instance or entirely manual setup.

**Reply by ionurboz** [reply]
I will try to change it with global instance. Thanks
